content/pt-br/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -0,0 +1,361 @@
+Para 2 ReplicaSets definindo o mesmo `.spec.selector` mas diferentes campos de `.spec.template.metadata.labels` e `.spec.template.spec`, cada ReplicaSet ignorará os Pods criados pelo outro ReplicaSet.
+{{< /note >}}
+
+### Replicas
+
+Você pode definir quantos Pods devem executar simultaneamente determinando `.spec.replicas`. O ReplicaSet irá criar/deletar os Pods para igualar à esse número.
+
+Se você não especificar o `.spec.replicas`, seu padrão é 1. 
+
+## Trabalhando com ReplicaSets
+
+### Deletando um ReplicaSet e seus Pods
+
+Para deletar um ReplicaSet e todos os seus Pods, use [`kubectl delete`](/docs/reference/generated/kubectl/kubectl-commands#delete). O [Garbage collector](/docs/concepts/workloads/controllers/garbage-collection/) automaticamente deleta todos os Pods dependentes por padrão.
+
+Quando usar a API REST ou a biblioteca `client-go`, você precisa definir `propagationPolicy` para `Background` ou `Foreground` na opção -d.
+Por exemplo:
+```shell
+kubectl proxy --port=8080
+curl -X DELETE  'localhost:8080/apis/apps/v1/namespaces/default/replicasets/frontend' \
+> -d '{"kind":"DeleteOptions","apiVersion":"v1","propagationPolicy":"Foreground"}' \
+> -H "Content-Type: application/json"
+```
+
+### Deletando apenas o ReplicaSet
+
+Você consegue deletar um ReplicaSet sem afetar qualquer um dos Pods usando [`kubectl delete`](/docs/reference/generated/kubectl/kubectl-commands#delete) com a opção `--cascade=orphan`.
+Quando usar a API REST ou a biblioteca `client-go`, você precisa definir `propagationPolicy` para `Orphan`.
+Por exemplo:
+```shell
+kubectl proxy --port=8080
+curl -X DELETE  'localhost:8080/apis/apps/v1/namespaces/default/replicasets/frontend' \
+> -d '{"kind":"DeleteOptions","apiVersion":"v1","propagationPolicy":"Orphan"}' \
+> -H "Content-Type: application/json"
+```
+
+Quando o original for deletado, você pode criar um novo ReplicaSet para substituí-lo. Contanto que o `.spec.selector` do velho e do novo sejam o mesmo, o novo irá adquirir os Pods antigos. Porém, o ReplicaSet não fará nenhum esforço nos Pods existentes para ajustá-los caso surja um novo e diferente template de pod.
+Para atualizar esses Pods para um novo spec de um modo controlado, use um [Deployment](/docs/concepts/workloads/controllers/deployment/#creating-a-deployment), já que ReplicaSets não suportam um rolling update diretamente.
+
+### Isolando Pods de um ReplicaSet
+
+Você pode remover Pods de um Replicaset trocando suas labels. Essa técnica pode ser usada para remover Pods de um service para depuramento, recuperação de dados, etc. Pods que forem removidos por esse método serão substituídos imediatamente (assumindo que o número de replicas não tenha sido alterado).
+
+### Escalando um ReplicaSet
+
+Um ReplicaSet pode ser facilmente escalado ou decaído simplismente atualizando o campo de `.spec.replicas`. O Replicaset controller garante que o número desejado de Pods com um seletor de label correspondente estejam disponíveis e operando.
```

Comment:
Minor:
* Replace "escalado ou decaído" with:
  * "escalonado para cima ou para baixo"; or
  * "redimensionado com mais ou menos réplicas".
  * This also applies to the next paragraph.

* Typo in "simplismente".
* "ReplicaSet controller" instead of "Replicaset controller".